### ISSUE TITLE
>npm install reanimated-bottom-sheet-navigator reanimated-bottom-sheet

### ISSUE THREAD

**Issue by daveasiamah** [osdnk/reanimated-bottom-sheet-navigator#1]
>npm install reanimated-bottom-sheet-navigator reanimated-bottom-sheet
When I tried the above I got the following error:
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/reanimated-bottom-sheet-navigator - Not found
npm ERR! 404
npm ERR! 404  'reanimated-bottom-sheet-navigator@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

A check at npmjs.com/package/reanimated-bottom-sheet-navigator returned not found as well.

Please help fix this issue.

**Comment by Twisterking** [comment]
+1, seems to be broken / not published to npm!

**Comment by rueiwoqp** [comment]
+1

**Comment by grifotv** [comment]
While this hasn't been published to NPM, you can add this to your `package.json` dependencies:
```json
"reanimated-bottom-sheet-navigator": "github:osdnk/reanimated-bottom-sheet-navigator",
```

**Comment by rueiwoqp** [comment]
> While this hasn't been published to NPM, you can add this to your `package.json` dependencies:
> 
> ```json
> "reanimated-bottom-sheet-navigator": "github:osdnk/reanimated-bottom-sheet-navigator",
> ```

That's true, but it would be nice to have it in npm and also would be nice to know if it's going to be supported and updated or if it's a fend for yourself situation.

**Comment by jedi2209** [comment]
Or you can use this command to add with yarn

`yarn add osdnk/reanimated-bottom-sheet-navigator`
